src/frontend/Komponenter/Oppgavebenk/kortNedBeskrivelse.ts

```diff
@@ -0,0 +1,18 @@
+export const kortNedBeskrivelse = (beskrivelse?: string) => {
+    if (!beskrivelse) {
+        return '';
```

Comment:
Jeg ville plassert denne metoden i `utils.ts`-filen som ligger i samme mappe. Den er tiltenkt at skal inneholde frittstående utils-metoder, slik som denne, som brukes i `Oppgavebenk`-mappen

Reply:
Kan man endre metodenavnet til `kortNedOppgavebeskrivelse` for å gjøre navnet mer spesifikt?

Reply:
Gode forslag, jeg endrer